packages/react-components/src/layout/src/Flex.tsx

```diff
@@ -141,7 +109,7 @@ function useIsGapSupported(noGap) {
 }
 
 export function InnerFlex({
-    direction,
+    direction = "row",
```

Comment:
I added a default value, instead of doing the || "row" where we needed it. It also fixes an issue i had with the flexdirection property

Reply:
I am not sure but I might haven't set a default value on purpose to because when it's not provided I might have wanted to have undefined here:

`flexDirection: direction && `${direction}${reverse ? "-reverse" : ""}`,`

Reply:
There were some tricky cases with this code.